scripts/eval_calc_errors.py

```diff
@@ -276,6 +281,25 @@ def calculate_errors_per_image(im_metaData):
                 depth_im = inout.load_depth(depth_path)
                 depth_im *= scene_camera[im_id]["depth_scale"]  # Convert to [mm].
 
+            # Create im_targets directly from scene_gt and scene_gt_info.
+            im_targets = {}
```

Comment:
im_targets creation is repeated in eval_calc_errors.py, eval_calc_errors_gpu.py and eval_calc_scores.py, can we refactor?